### ISSUE TITLE
Export command does not export markers

### ISSUE THREAD

**Issue by michaljelonek** [sdispater/poetry#1011]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: MacOS 10.14.3
- **Poetry version**: latest, 1.0.0a2
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: 
```
[tool.poetry]
name = "export"
version = "0.1.0"
description = ""
authors = ["Michał Jelonek <mjelonek92@gmail.com>"]

[tool.poetry.dependencies]
python = ">=3.5, <3.8"
boto3 = {version = "^1.9",platform = "darwin"}

[tool.poetry.dev-dependencies]
pytest = "^3.0"

[build-system]
requires = ["poetry>=0.12"]
build-backend = "poetry.masonry.api"
```

## Issue

The export command does not export markers. Is this intentional?

The lock file has correct markers:
```
[[package]]
category = "main"
description = "The AWS SDK for Python"
marker = "sys_platform == \"darwin\""
name = "boto3"
optional = false
python-versions = "*"
version = "1.9.127"
```

However after exporting, the requirements.txt file doesn't have them (`poetry export --without-hashes -f requirements.txt`):
```
boto3==1.9.127
botocore==1.12.127
docutils==0.14
jmespath==0.9.4
python-dateutil==2.8.0
s3transfer==0.2.0
six==1.12.0
urllib3==1.24.1
```

**Comment by danields761** [comment]
Yeah, i'am agree with @michaljelonek, that markers should be exported correctly according to PEP-440, i suppose

**Comment by sdispater** [comment]
Fixed in #1277
